src/modules/deal-folder/dto/upload-file-in-deal-folder-request.dto.ts

```diff
@@ -33,9 +33,9 @@ export class UploadFileInDealFolderRequestItem {
 
   @ValidatedStringApiProperty({
     description: 'The path of the location/folder in Azure storage from which the file needs to be loaded.',
-    minLength: 1,
+    minLength: 24,
     maxLength: 250,
-    pattern: /^[\w\-:/\\()\s]+$/,
+    pattern: /^[a-fA-F\d]{24}[\w\-:/\\()\s]*$/,
```

Comment:
I think we should write a comment to explain why `[a-fA-F\d]{24}` is in the pattern here, it'd be quite surprising to a reader otherwise

Also does it matter that we don't force a `/` after the first 24 characters if the string has length more than 24?

Reply:
Added comment explaining `[a-fA-F\d]{24}` part. Good point about starting with a `/` after the hex, I've adjusted the regex (and the generator function) to ensure that